openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/FunctionReturnTypeGenerator.java

```diff
@@ -352,7 +352,7 @@ null, createIdentifierToken("public type"),
      * @param typeName    - Given Node name
      * @param typeDefNode - Generated Node
      */
-    public void updateTypeDefinitionNodeList(String typeName, TypeDefinitionNode typeDefNode) {
+    private void updateTypeDefinitionNodeList(String typeName, TypeDefinitionNode typeDefNode) {
```

Comment:
Is it possible to move this function as common util to Util class since this function has duplicated in several place ?

Reply:
Added. Please review